packages/Algernon.package/ALGCache.class/instance/clear.st

```diff
@@ -1,4 +1,3 @@
 caching
 clear
```

Comment:
Do we need this function ? As this ist just an "rename" for removeAll, we might want to call removeAll instead of clear.

Reply:
This is because we need to ensure that old calls from `ALGInstallation instance cleanup` need work even if the new code has been loaded.

Reply:
Okay, I see.